tests/catch/unit/rtc/includepath.cc

```diff
@@ -0,0 +1,128 @@
+#include <hip_test_common.hh>
+
+#include <hip/hiprtc.h>
+#include <hip/hip_runtime.h>
+
+#include <cassert>
+#include <cstddef>
+#include <memory>
+#include <iostream>
+#include <iterator>
+#include <vector>
+#include <fstream>
+
+static constexpr auto NUM_THREADS{128};
+static constexpr auto NUM_BLOCKS{32};
+
+// This test verifies hiprtc compilation by passing include path option using -I with spaces
+// before the path. eg: -I   ../ or -I /path/to/headers etc.
+
+TEST_CASE("Unit_hiprtc_includepath") {
+  using namespace std;
+
+  string saxpy = "";
+  {
+    fstream f("saxpy.h");
+    if (f.is_open()) {
+      size_t sizeFile;
+      f.seekg(0, fstream::end);
+      size_t size = sizeFile = (size_t)f.tellg();
+      f.seekg(0, fstream::beg);
+      saxpy.resize(size, ' ');
+      f.read(&saxpy[0], size);
+      f.close();
+   }
+ }
+
+  hiprtcProgram prog;
+  hiprtcCreateProgram(&prog,         // prog
+                      saxpy.c_str(), // buffer
+                      "saxpy.cu",    // name
+                      0, nullptr, nullptr);
+
+  hipDeviceProp_t props;
+  int device = 0;
+  HIP_CHECK(hipGetDeviceProperties(&props, device));
+#ifdef __HIP_PLATFORM_AMD__
+  string sarg = string("--gpu-architecture=") + props.gcnArchName;
+#else
+  string sarg = string("--gpu-architecture=compute_")
+    + to_string(props.major) + to_string(props.minor);
+#endif
+  // Need to find the header files from the include path
+  // It is set to headers in the current directory here
+  const char* options[] = {
+      sarg.c_str(), "-I    ./headers"
```

Comment:
@mangupta @gargrahul I want to change this line to "-I./headers" since the spaces in the include path are not accepted by clang/hiprtc.  But I'm unable to do so. Is it possible to edit the PR?

Reply:
@satyanveshd : done. is any other change required?

Reply:
@mangupta Thanks. This is fine. Once the CI passes, it can be merged.